packages/yourloops/components/menus/patient-team-settings-menu.tsx

```diff
@@ -154,18 +137,10 @@ function TeamMenu(): JSX.Element {
     }
   }
   return (
-    <React.Fragment>
-      <Box
-        id="team-menu"
-        data-testid="team-menu"
-        display="flex"
-        role="button"
-        alignItems="center"
-        className={clickableMenu}
-        onClick={event => {
-          setAnchorEl(event.currentTarget)
-        }}
-      >
+    <>
+      <IconButton id="team-menu" color="inherit" onClick={event => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏We should have a `Button` here instead of IconButton because the ripple effect becomes ugly if there is more than one icon. With Button component ripple effect is good with proper shape !